### DISCUSSION TITLE
e-factura

### DISCUSSION THREAD

**Discussion by adbadu** [woocommerce/woocommerce#50576] [Checkout Flow]
pentru integrare cu e-factura este necesar campul CNP si sector, care nu sunt in blocul finalizare. cum pot sa le adaug?

> translation: for integration with the e-invoice, the CNP and sector fields are required, which are not in the finalization block. how can i add them?

**Comment by senadir** [comment]
You can add fields using our Additional fields API

https://github.com/woocommerce/woocommerce/blob/release/9.2/plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/additional-checkout-fields.md

**Reply by adbadu** [reply]
page not found when i click

**Reply by senadir** [reply]
Updated the link

**Reply by adbadu** [reply]
thank you. and where do i put this code?

**Reply by senadir** [reply]
In your functions.php or a snippet plugin.
